tests/integration/Tribe/Events/Integrations/QR/RedirectionsTest.php

```diff
@@ -0,0 +1,204 @@
+<?php
+
+namespace Tribe\Events\Integrations\QR;
+
+use Codeception\TestCase\WPTestCase;
+use TEC\Events\QR\Redirections;
+use TEC\Events\QR\Routes;
+use TEC\Events\QR\Settings;
+use Tribe__Events__Main as TEC;
+
+/**
+ * Tests QR Redirections functionality
+ *
+ * @group   core
+ * @group   qr
+ *
+ * @package TribeEvents
+ */
+class RedirectionsTest extends WPTestCase {
+
+	/**
+	 * The redirections instance.
+	 *
+	 * @var \TEC\Events\QR\Redirections
+	 */
+	protected $redirections;
+
+	/**
+	 * The routes instance.
+	 *
+	 * @var \TEC\Events\QR\Routes
+	 */
+	protected $routes;
+
+	/**
+	 * The test event ID.
+	 *
+	 * @var int
+	 */
+	protected $test_event_id;
+
+	/**
+	 * The option slugs.
+	 *
+	 * @var array
+	 */
+	protected $slugs;
+
+	/**
+	 * Set up the test.
+	 *
+	 * @return void
+	 */
+	function setUp() {
+		parent::setUp();
+
+		// Get option slugs once
+		$this->slugs = Settings::get_option_slugs();
+
+		// Enable QR
+		tribe_update_option( $this->slugs['enabled'], true );
+
+		// Register the routes
+		$this->routes = tribe( Routes::class );
+		$this->routes->do_register();
```

Comment:
isn't do_register a protected method ? register is what we should be calling instead i believe.